### DISCUSSION TITLE
compiling problem

### DISCUSSION THREAD

**Discussion by crislikesanimation** [KadeArchive/Kade-Engine#1111] [General]
when I compile it shows this 

source/PlayState.hx:3379: characters 17-32 : Class<webm.WebmPlayer> has no field SKIP_STEP_LIMIT
/usr/local/lib/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
/usr/local/lib/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access

I'm compiling on Mac but this never happens and now it is

**Comment by MadeySlime** [comment]
I had the same problem, you have to go in PlayState.hx, find the code (WebmPlayer.SKIP_STEP_LIMIT = 90;) at the line 3379 and delete it ! then save the hx file and try compiling

**Reply by EcoVeni** [reply]
person above me will fix your problem just listen to him

**Reply by crislikesanimation** [reply]
> I had the same problem, you have to go in PlayState.hx, find the code (WebmPlayer.SKIP_STEP_LIMIT = 90;) at the line 3379 and delete it ! then save the hx file and try compiling

THANK U U HELPED SO MUCH

**Reply by WebosSan** [reply]
> I had the same problem, you have to go in PlayState.hx, find the code (WebmPlayer.SKIP_STEP_LIMIT = 90;) at the line 3379 and delete it ! then save the hx file and try compiling

hmmm so if i do that the ge
